.vsconfig

```diff
@@ -0,0 +1,6 @@
+﻿{
+  "version": "1.0",
+  "components": [
+    "Microsoft.VisualStudio.Workload.ManagedGame"
+  ]
+}
```

Comment:
What is this file for? Do we need to CM it? Looks like a visual studio configuration file or something.